core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/DiagnosticLogUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.utils;
```

Comment:
missing license header

Reply:
This has been fixed now (Removed the Util class).